__tests__/index.spec.tsx

```diff
@@ -48,8 +49,11 @@ it('should render posts', async () => {
   // are solved:
   // https://github.com/kentcdodds/react-testing-library/pull/216
   // https://github.com/facebook/react/issues/14050
-  render(null as any)
-  await wait(0)
+  // render(null as any)
```

Comment:
If this hack is not needed anymore, could you maybe remove the whole block of code?

Reply:
I commented these lines by accident, they are uncommented now